.travis.yml

```diff
@@ -9,8 +9,7 @@ python:
     - pypy
     - pypy3
 install:
-    - pip install .
-    - pip install zope.keyreference[test]
+    - pip install .[test]
```

Comment:
So that didn't solve the problem :(

On older python's, you need to first `pip install -U pip`. I forgot to mention that. Which nowadays will also update setuptools.